storybook/pages/JoinCommunityPermissionsEditor.qml

```diff
@@ -125,6 +125,16 @@ ColumnLayout {
             text: "Long model"
             onCheckedChanged: if(checked) root.communityHoldingsModel = PermissionsModel.longPermissionsModel
         }
+
+        RadioButton {
+            text: "Private met model"
```

Comment:
I am not sure what is a `Private met model` ?

Reply:
It means a model where all the permission rows are private and at least one is met